kyuubi-server/pom.xml

```diff
@@ -245,10 +257,6 @@
                     <groupId>org.apache.hadoop</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
-                <exclusion>
-                    <groupId>org.apache.hive</groupId>
-                    <artifactId>*</artifactId>
```

Comment:
We manage hive dependency manually instead of using maven transitive dependencies, because the hive dependencies is likely out of control if we do that. Please restore this exclusion and add everything explicitly.

PS: this rule only apply to hive, in most cases, we prefer use transitive dependencies.

Reply:
done